hazelcast/src/test/java/com/hazelcast/jet/core/JetTestSupport.java

```diff
@@ -303,6 +305,20 @@ public static long assertJobRunningEventually(HazelcastInstance instance, Job jo
         return executionId;
     }
 
+    /**
+     * Asserts that the {@code job} is eventually SUSPENDED and waits until the suspension cause is set.
+     */
+    public static void assertJobSuspendedEventually(Job job) {
+        assertJobStatusEventually(job, SUSPENDED);
+        assertTrueEventually(() -> {
```

Comment:
Maybe we can keep the comment from here

https://github.com/hazelcast/hazelcast-enterprise/pull/5997/files#diff-b56060e4b06be6849511295881f03b70c58081371ecf33bb428e4188b196d0e0L354-L356

?